packages/dbos-cloud/package.json

```diff
@@ -11,7 +11,10 @@
   "main": "cli.ts",
   "homepage": "https://docs.dbos.dev/",
   "scripts": {
-    "build": "tsc --project tsconfig.json && chmod +x ./dist/packages/dbos-cloud/cli.js",
+    "build": "tsc --project tsconfig.json && npm run chmod",
```

Comment:
Do we actually need this?  Why is there a `chmod` for these packages but not for the main package?

Reply:
It allows the .js file to be run off the command line.  Not every package has a runnable .js file.  If we don't need to execute it that way (say, if we only use npx scripts), this could be skipped entirely.

I don't think I invented this approach though, but rather copied from somewhere else.